src/backend/common/cache_clearing/get_affected_queries.py

```diff
@@ -345,5 +346,20 @@ def insight_updated(affected_refs: TAffectedReferences) -> List[TCacheKeyAndQuer
     for year in years:
         queries.append(insight_query.InsightsLeaderboardsYearQuery(year))
         queries.append(insight_query.InsightsNotablesYearQuery(year))
+        for district_abbreviation in RenamedDistricts.get_latest_codes():
```

Comment:
I think this ends up over-clearing caches. I think `_affected_references` should include `district_abbreviation` (and remove it from `_mutable_attrs`).